pygmm/bayless_abrahamson_2019.py

```diff
@@ -165,7 +165,8 @@ def _calc_site_response(self, ln_eas_ref: float) -> np.ndarray:
 
         f_s = f_sl + f_nl
 
-        if v_s30 < 800:
+        flag_test = False
```

Comment:
Is this needed?

Reply:
The condition `v_s30 < 800` was in the original script. I added `flag_test = False` to suppress the plotting if it is not needed. Alternatively, lines 168 to 172 can be deleted or kept in as is in the main branch.